_posts/2018-11-15-using-vault-with-strimzi.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title:  "Using the Vault PKI secrets engine to generate a CA to use with the Strimzi Kafka Operator"
```

Comment:
What about "... to generate a CA certificate ..." or "... to setup a CA to use ..." ?